frontend/src/components/body/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,21 @@
+import {  useEffect, useState } from "react";
```

Comment:
1. Take default value as a prop.
    2. Make type of data store as value generic.
    3. Incorrect return type of getItemFromLocalStorage and use undefined instead of null - [Article](https://medium.com/@hbarcelos/why-i-banned-null-from-my-js-code-and-why-you-should-too-13df90323cfa)